doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst

```diff
@@ -411,40 +384,19 @@ The band lock mask allows you to set the bands on which you want the modem to op
 Each bit in the :kconfig:option:`CONFIG_LTE_LOCK_BAND_MASK` option represents one band.
 The maximum length of the string is 88 characters (bit string, 88 bits).
 
-The band lock is a non-volatile setting that must be set before activating the modem.
-It disappears when the modem is reset.
-To prevent this, you can set the modem in *power off* mode, by either:
-
-* Sending the AT command ``AT+CFUN=0`` directly.
-* Calling the :c:func:`lte_lc_power_off` function while the *LTE Link Control Library* is enabled.
-
-Both these options save the configurations and historical data in the Non-Volatile Storage before powering off the modem.
-
-As a recommendation, turn off the band lock after the connection is established and let the modem use the historical connection data to optimize the network search, in case the device is disconnected or moved.
-
 For more detailed information, see the `band lock section in the AT Commands reference document`_.
 
 .. _nrf9160_ug_network_mode:
 
-Network mode
-************
-
-The modem supports LTE-M (Cat-M1) and Narrowband Internet of Things (NB-IoT or LTE Cat-NB).
-By default, the modem starts in LTE-M mode.
-However, this is highly configurable.
-
-When using the LTE Link Control driver, you can select LTE-M with :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M` or NB-IoT with :kconfig:option:`CONFIG_LTE_NETWORK_MODE_NBIOT`.
-
-To start in NB-IoT mode without the driver, send the following command before starting the modem protocols (by using ``AT+CFUN=1``)::
-
-   AT%XSYSTEMMODE=0,1,0,0
+System mode
+***********
 
-To change the mode at runtime, set the modem to LTE RF OFF state before reconfiguring the mode, then set it back to normal operating mode::
+The modem system mode configuration is used to select which of the supported systems, :term:`LTE-M`, :term:`NB-IoT<Narrowband Internet of Things (NB-IoT)>` and :term:`GNSS<Global Navigation Satellite System (GNSS)>`, are enabled.
```

Comment:
Either
```suggestion
You can configure the system mode of the modem to select which of the supported systems, :term:`LTE-M`, :term:`NB-IoT<Narrowband Internet of Things (NB-IoT)>` and :term:`GNSS<Global Navigation Satellite System (GNSS)>`, are enabled.
```
Unless "system mode configuration" is a thing in itself, in which case:
```suggestion
The system mode configuration of the modem is used to select which of the supported systems, :term:`LTE-M`, :term:`NB-IoT<Narrowband Internet of Things (NB-IoT)>` and :term:`GNSS<Global Navigation Satellite System (GNSS)>`, are enabled.
```

Reply:
The latter one sounds better to me, I'll use that.